odl/solvers/util/callback.py

```diff
@@ -440,13 +440,20 @@ def __init__(self, *args, **kwargs):
             where ``cur_iter_num`` is the current iteration number.
         step : positive int, optional
             Number of iterations between plots. Default: 1
-        saveto : string, optional
+        saveto : string or callable, optional
             Format string for the name of the file(s) where
-            iterates are saved. The file name is generated as
+            iterates are saved.
+
+            If saveto is a string, the file name is generated as::
```

Comment:
\`\`saveto\`\`

Leave a whitespace between "as" and "::", then the colon is not printed.